wagtail_wordpress_import/handle_inline_shortcodes.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+
+class InlineShortcodeHandler:
+    """Subclasses should declare a shortcode_name and provide
+    a construct_html_tag method for converting the shortcode to a HTML tag.
+    """
+
+    inline_shortcode_name: str
+
+    element_name = "span"
+
+    def __init__(self):
+        # Subclasses should declare a shortcode_name
+        if not hasattr(self, "shortcode_name"):
+            raise NotImplementedError(
+                "Create a subclass of BlockShortcodeHandler with a shortcode_name attribute"
+            )
+        pattern = re.compile(r"^\S[a-zA-Z0-9_\S]+\S$")
+        # shortcode_name must use upper or lower case letters or digits and cannot contain spaces
+        if not re.match(pattern, self.shortcode_name):
+            raise ValueError(
+                "The shortcode_name attribute must use upper or lower case letters or digits and cannot contain spaces"
+            )
```

Comment:
**Suggestion**: You can use `if not pattern.match(self.shortcode_name)` here.

Reply:
Thanks for this. I will update it.

It's also the same in `class BlockShortcodeHandler:` in `wagtail_wordpress_import/prefilters/handle_shortcodes.py`

I've not updated that here but will do if you think I should.

Reply:
Sure. It's not really an issue. I was just sharing for the sake of spreading awareness.